spec/controllers/phylo_trees_controller_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe PhyloTreesController, type: :controller do
+  describe "GET validate_name" do
+    before do
+      joe = create(:joe)
+      sign_in joe
+
+      create(:phylo_tree, name: "existing_name")
+    end
+
+    context "with unique name" do
+      it "is valid" do
+        name = "unique_name"
+
+        get :validate_name, params: { format: "json", name: name }
+
+        json_response = JSON.parse(response.body)
+        puts json_response
```

Comment:
I'm assuming you meant to remove the `puts`

Reply:
Thanks!